testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -595,6 +603,11 @@ QUnit.test("popup height can be changed according to the content if height = aut
 });
 
 QUnit.test("popup height should support top and bottom toolbars if height = auto", assert => {
+    if(isIE11) {
```

Comment:
Skipping a test in IE11 without provviding a new one means untested behavior in this browser